### PR TITLE
Migrated lazy loading guidance doc from Google Docs to markdown.

### PR DESCRIPTION
**Pull request by malchata**
Wrote an article on lazy loading images and video, after iterating on feedback.

Let me know what you all think of it, and what feedback you have!

### CODE REVIEW

File: src/content/en/fundamentals/performance/lazy-loading-guidance/images-and-video/index.md [hunk 1]
```diff
@@ -0,0 +1,699 @@
...
+
+{# wf_updated_on: 2018-03-23 #}
+{# wf_published_on: 2018-04-04 #}
+{# wf_blink_components: Blink>PerformanceAPIs,Blink>JavaScript>Runtime,Blink>Input #}
+
+# Lazy Loading Images and Video {: .page-title }
+
+{% include "web/_shared/contributors/jeremywagner.html" %}
+
+The portion of [images](http://beta.httparchive.org/reports/state-of-images?start=earliest&end=latest) and [video](http://beta.httparchive.org/reports/page-weight#bytesVideo) in per-page payloads of typical websites can be significant. Unfortunately, project stakeholders may be unwilling to cut any media resources from their existing applications. Such impasses are frustrating, especially when all parties involved want to improve site performance, but can't agree on how to get there. Fortunately, lazy loading is a solution that lowers initial page payload _and_ load time, but doesn't skimp on content.
```
Comment (addyosmani) [inline src/content/en/fundamentals/performance/lazy-loading-guidance/images-and-video/index.md]: I'd consider simplifying or rewording the opening sentence (hard to parse on re-reading). Perhaps, the portion of images and video in the typical payload of a website can be significant?

File: src/content/en/fundamentals/performance/lazy-loading-guidance/images-and-video/index.md [hunk 2]
```diff
@@ -0,0 +1,699 @@
...
+by using event handlers such as `scroll` or `resize`. While this approach is the
+most compatible across browsers, modern browsers offer a more performant and
+efficient way to do the work of checking element visibility via [the
+intersection observer
+API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API).
+
+Note: Intersection observer is not supported in all browsers. If compatibility
+across browsers is crucial, be sure to read [the next section](#TODO), which
+shows you how to lazy load images using less performant (but more compatible!)
+scroll and resize event handlers.
```
Comment (addyosmani) [inline src/content/en/fundamentals/performance/lazy-loading-guidance/images-and-video/index.md]: Note that there's a stray TODO here.

File: src/content/en/fundamentals/performance/lazy-loading-guidance/images-and-video/index.md [hunk 3]
```diff
@@ -0,0 +1,699 @@
...
+populated with URLs to the desired image content.
+
+#### Using intersection observer
+
+If you've written lazy loading code before, you may have accomplished your task
+by using event handlers such as `scroll` or `resize`. While this approach is the
+most compatible across browsers, modern browsers offer a more performant and
+efficient way to do the work of checking element visibility via [the
+intersection observer
+API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API).
```
Comment (addyosmani) [inline src/content/en/fundamentals/performance/lazy-loading-guidance/images-and-video/index.md]: It might be useful to contextually link to https://developers.google.com/web/updates/2016/04/intersectionobserver somewhere here.

File: src/content/en/fundamentals/performance/lazy-loading-guidance/images-and-video/index.md [hunk 4]
```diff
@@ -0,0 +1,699 @@
...
+- [lozad.js](https://github.com/ApoorvSaxena/lozad.js) is a super lightweight
+option that uses intersection observer only. As such, it's highly performant,
+but will need to be polyfilled before you can use it on older browsers.
+- [blazy](https://github.com/dinbror/blazy) is another such option that bills
+itself as a lightweight lazy loader (weighing in at 1.4 KB). As with lazysizes,
+it doesn't need any third party utilities to load, and works for IE7+.
+Unfortunately, it doesn't use intersection observer.
+- [yall.js](https://github.com/malchata/yall.js) is a library I wrote that uses
+IntersectionObserver and falls back to event handlers. It's compatible with IE11
+and major browsers.
```
Comment (addyosmani) [inline src/content/en/fundamentals/performance/lazy-loading-guidance/images-and-video/index.md]: I also just came across https://github.com/jasonslyvia/react-lazyload which appears popular in the React community. @StephenFluin is there a canonical library Angular recommends for lazy loading media right now?

File: src/content/en/fundamentals/performance/lazy-loading-guidance/images-and-video/index.md (hunk 2 above)
Comment (malchata) [reply src/content/en/fundamentals/performance/lazy-loading-guidance/images-and-video/index.md]: Ah, I had meant replace that with in-page links. Will do.

**Comment by malchata** [comment]
Got these changes addressed @addyosmani! Let me know if you (or others) have any other feedback.

-j
